### DISCUSSION TITLE
Neural Network model

### DISCUSSION THREAD

**Discussion by ravi7522** [ravi7522/Football-Prediction#20] [General]
![image](https://github.com/ravi7522/Football-Prediction/assets/23443691/7d0b08a2-ae37-4b57-9b96-74ebf0358bbf)

This is the result of Neural Network I created for the prediction.

Accuracy: 0.5153964588144727
Precision: 0.4713969628736005
Recall: 0.5153964588144727
F1 Score: 0.4704386072157805

Performance of the model. 

What are all of your thoughts on this ?

**Comment by babuaditya97** [comment]
looks decent to me, we still can see if we can pull the scores a bit higher. I will try to increase the complexity.
